gym/wrappers/test_flatten_observation.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+
+import gym
+from gym.wrappers import FlattenObservation
+
+
+@pytest.mark.parametrize('env_id', ['Pong-v0', 'SpaceInvaders-v0'])
```

Comment:
same spiel about atari envs in tests as in other wrapper PRs applies here as well